library/src/main/java/com/mux/player/media/MediaItems.kt

```diff
@@ -136,7 +162,22 @@ enum class PlaybackResolution {
   FOUR_K_2160,
 }
 
+/**
+ * The order of preference for adaptive streaming.
+ *
+ * version will be tried first, switching toward higher ones. [Descending] means the opposite,
```

Comment:
Did this get truncated?

Reply:
Woah yeah that looks weird. I'll check it out